onedal/common/_policy.py

```diff
@@ -72,3 +76,12 @@ def __init__(self, queue):
                     super().__init__(self._queue.sycl_device.get_filter_string())
                     return
             super().__init__(self._queue.addressof_ref())
+
+    try:
+        class _SPMDDataParallelInteropPolicy(_backend.spmd_data_parallel_policy):
+            def __init__(self, queue):
+                self._queue = queue
+                super().__init__(self._queue)
+    except:
```

Comment:
Add explicit error type

Reply:
Fixed by moving to another file. Anyway it is private functionality